src/app/js/fields/FieldComposedOf.js

```diff
@@ -173,12 +168,30 @@ const FieldComposedOf = ({
     scope,
     subresourceId,
 }) => {
-    const autocompleteValue = columns.map(column => {
-        return fields.find(field => field.name === column);
-    });
+    const options = useMemo(() => {
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Explain the aim of the block

To be able to see deleted field